src/plugins/vis_type_timeseries/common/get_last_value.test.js

```diff
@@ -37,8 +37,4 @@ describe('getLastValue(data)', () => {
       ])
     ).toBe('-');
   });
-
-  test('should allows to override the default value', () => {
```

Comment:
agree, we don't have any cases where we do it.

Reply:
It has a typo too 😛